src/Microsoft.Health.Dicom.Web/Startup.cs

```diff
@@ -52,17 +55,20 @@ public virtual void Configure(IApplicationBuilder app)
     }
 
     /// <summary>
-    /// Adds ApplicationInsights for telemetry and logging.
+    /// Adds Open telemetry exporter for Azure monitor.
     /// </summary>
-    private void AddApplicationInsightsTelemetry(IServiceCollection services)
+    private void AddOpenTelemetry(IServiceCollection services)
     {
         string instrumentationKey = Configuration["ApplicationInsights:InstrumentationKey"];
 
         if (!string.IsNullOrWhiteSpace(instrumentationKey))
         {
             var connectionString = $"InstrumentationKey={instrumentationKey}";
 
-            services.AddApplicationInsightsTelemetry(aiOptions => aiOptions.ConnectionString = connectionString);
```

Comment:
We were using AppInsights for logs. Are we planning to use Open Telemetry for logs here too?

Reply:
Yes, the plan is to move logs also to Open telemetry. I haven't removed any of the manual logging statements. Are there any additional logging from the AppInsights sdk?

Reply:
I thought this statement registered an `ILogger` that logged to app insights

Reply:
Added the function back.